app/views/summaries/_summary.html.erb

```diff
@@ -30,15 +30,15 @@
 
 <div class="page-section">
   <h3><%= sul_icon :'sharp-attach_money-24px', classes: 'lg' %> Fines &amp; fees payable: <%= number_to_currency(patron.fines.sum(&:owed)) %></h3>
-
-  <% if params[:group] && patron.fines.sum(&:owed).positive? # temporary fix until Symphony CGI scripts are updated %>
-    <% if patron.sponsor? # temporary fix until Symphony CGI scripts are updated %>
+  <!-- TODO: after FOLIO launch remove this condition -->
```

Comment:
Make this an ERB comment so that it doesn't actually show up in the public source.